FastSurferCNN/data_loader/conform.py

```diff
@@ -95,75 +93,91 @@ def make_parser() -> argparse.ArgumentParser:
+        const="min",
+        help="Specifies whether the input is or should be conformed to the minimal voxel size (used for high-res "
+             "processing) - overwrites --vox_size.",
+    )
+    parser.add_argument(
+        "--img_size",
+        dest="img_size",
+        default="auto",
+        type=__img_size,
+        help="Specifies the image size to conform to (deactivated by 'any'). Also allows 'fov' for conforming to the "
+             "field of view.",
+    )
+    parser.add_argument(
+        "--rescale",
+        dest="rescale",
+        default="auto",
+        choices=("auto", "robust", "none"),
+        help="Specifies whether image intensities should be rescaled, 'robust' will robustly intensities to the output "
+             "rescale dtype range, e.g. 0-255 for uint8/uchar, 'none' will not change intensities, 'auto' will only "
+             "robust-ly rescale for data type uint8 (which is the default).",
     )
     advanced = parser.add_argument_group("Advanced options")
     advanced.add_argument(
         "--conform_to_1mm_threshold",
         type=__conform_to_one_mm,
-        help="Advanced option to change the threshold beyond which images are "
-             "conformed to 1 (default: infinity, all images are conformed to their "
-             "minimum voxel size).",
+        help="Advanced option to change the threshold beyond which images are conformed to 1 (default: infinity, "
+             "all images are conformed to their minimum voxel size).",
     )
     advanced.add_argument(
         "--dtype",
         dest="dtype",
         default="uint8",
         type=__target_dtype,
-        help="Specifies the target data type of the target image or 'any' (default: "
-             "'uint8', as in FreeSurfer)",
+        help="Specifies the target data type of the target image or 'any' (default: 'uint8', as in FreeSurfer)",
     )
     advanced.add_argument(
-        "--no_strict_lia",
-        dest="force_strict_lia",
-        action="store_false",
-        help="Ignore the forced LIA reorientation.",
+        "--orientation",
+        dest="orientation",
+        default="lia",
+        choices=("lia", "soft-lia", "soft lia", "native"),
+        help="Specify the target data orientation."
     )
     advanced.add_argument(
-        "--no_lia",
-        dest="force_lia",
-        action="store_false",
-        help="Ignore the reordering of data into LIA (without interpolation). "
-             "Supersedes --no_strict_lia",
+        "--no_strict_lia", "--allow_soft_lia",
+        dest="orientation",
+        action="store_const",
+        const="soft lia",
+        help="Ignore the forced LIA reorientation and only reorient to a soft LIA orientation.",
     )
     advanced.add_argument(
-        "--no_iso_vox",
-        dest="force_iso_vox",
-        action="store_false",
-        help="Ignore the forced isometric voxel size (depends on --conform_min).",
+        "--no_lia", "--allow_native",
```

Comment:
and another lia flag. I think this is very confusing.

Reply:
see above (I can add that this is legacy to the help text)